README.md

```diff
@@ -15,14 +15,43 @@ The image uses the following software:
 
 To use this project successfully, you should first decide a few things:
 
-First, decide whether you want your container to be part of the public, production Stellar network (referred to as the _pubnet_) or the test network (called testnet) that we recommend you use while developing software because you need not worry about losing money on the testnet. Additionally, we have added a standalone network (called standalone) which allows you to run your own private Stellar network. In standalone network mode, you can optionally pass `--protocol-version {version}` parameter to run a specific protocol version (defaults to latest version). You'll provide either `--pubnet`, `--testnet` or `--standalone` as a command line flag when starting the container to determine which network (and base configuration file) to use.
+First, decide whether you want your container to be part of the public, production Stellar network (referred to as the _pubnet_) or the test network (called testnet) that we recommend you use while developing software because you need not worry about losing money on the testnet. Additionally, we have added a standalone network (called standalone) which allows you to run your own private Stellar network.
```

Comment:
As this PR's purpose is to get from "0 to working code" as quickly as possible, maybe a link to any SDK, like go SDK for how to set the network passphrase would help (people can then look on their own, but at least they'll get the idea) ?

Reply:
Good idea. I added a note here: 9102197b3d42fe573370ea2699b62dab79ba8ab5. There are a lot of SDKs, and the Go SDK isn't the most popular. I added a note without example code. To add example code is out-of-scope I think, and the note will help enough. Let's see if that's sufficient.